src/ssl/cert_validate_message.cc

```diff
@@ -90,6 +91,22 @@ get_error_id(const char *label, size_t len)
 
 bool
 Ssl::CertValidationMsg::parseResponse(CertValidationResponse &resp)
+{
+    try {
+        tryParsingResponse(resp);
+        return true;
+    }
+    catch (...) {
+        debugs(83, DBG_IMPORTANT, "ERROR: Cannot parse sslcrtvalidator_program response:" <<
+               Debug::Extra << "problem: " << CurrentException);
+        return false;
```

Comment:
Here and in similar PR places, the try/catch blocks are stopping the propagation of code changes required for accommodating the now-throwing certificate reading API. As we improve callers, these blocks should be moved higher up and eventually be removed completely, but that usually requires non-trivial caller changes that are best done in dedicated PRs. Otherwise, the ensuing noise is likely to cause memory leaks and other problems.